includes/class-plugin-manager.php

```diff
@@ -16,6 +16,42 @@
  */
 class Plugin_Manager {
 
+	/**
+	 * Get info about all the managed plugins and their status.
+	 *
+	 * @todo Define what the structure of this looks like better and load it up from a config or something.
+	 *
+	 * @return array of plugins info.
+	 */
+	public static function get_managed_plugins() {
+		$managed_plugins = [
+			'jetpack' => [
+				'name'     => __( 'Jetpack', 'newspack' ),
+				'download' => 'wporg',
+			],
+			'amp'     => [
+				'name'     => __( 'AMP', 'newspack' ),
+				'download' => 'wporg',
+			],
+		];
```

Comment:
What sort of information do we need available about the plugins? I've kicked it off with the following info for each plugin:
```
name - The plugin name.
download - 'wporg' or a URL to a zip file.
status - 'active', 'inactive', or 'uninstalled'
```

Any additional information we need? Instead of having this information hard-coded in the `get_managed_plugins` method, is there a better way of storing/retrieving the managed plugin info?